app/models/profile.rb

```diff
@@ -18,15 +20,21 @@ class Profile < ActiveRecord::Base
   validates :name, :lastname, presence: true, on: :update
   validates :start_year, inclusion: { in: 1954..(Time.zone.today.year + 1) }, on: :update
 
+  scope :search_names, ->(name,lastname) { where('name LIKE ? AND lastname LIKE ?',"%#{name}%","%#{lastname}%")}
```

Comment:
Line is too long. [112/100]<br>Space missing after comma.<br>Space missing inside }.